test/core/api/transport/http_listener_test.cpp

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "core/api/transport/listener_test.hpp"
+
+#include "api/transport/impl/http/http_listener_impl.hpp"
+#include "core/api/client/http_client.hpp"
+
+using kagome::api::HttpListenerImpl;
+using test::HttpClient;
+
+using WsListenerTest = ListenerTest<HttpListenerImpl>;
+
+TEST_F(WsListenerTest, EchoSuccess) {
```

Comment:
please drop few comments explaining what this test does. Use given when then terminology (chech any test in the repo for test comments template)

Reply:
done